apps/cowswap-frontend/src/legacy/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -20,29 +20,50 @@ import { TokenAmount } from 'common/pure/TokenAmount'
 import CurrencyListMod, { StyledBalanceText, TagContainer } from './CurrencyListMod'
 import { Tag as TagMod } from './styled'
 
-const UNSUPPORTED_TOKEN_TAG = [
+
+const TOKEN_TAGS: TagInfo[] = [
```

Comment:
Add an enum for the tag type and transform this into a record

```ts
enum Tags = {
  UNSUPPORTED,
  GAS_FREE,
}

const TOKEN_TAGS: Record<Tags, TagInfo> = {
...
  [Tag.UNSUPPORTED] = {...},
}
```

Reply:
@alfetopito Thanks for the suggestion, updated please review!